shared/src/business/useCases/processStreamRecordsInteractor.js

```diff
@@ -119,22 +124,33 @@ exports.processStreamRecordsInteractor = async ({
         records: removeRecords,
       });
 
-      if (failedRecords.length) {
-        for (const failedRecord of failedRecords) {
-          try {
-            await applicationContext.getPersistenceGateway().deleteRecord({
-              applicationContext,
-              indexName: failedRecord['_index'],
-              recordId: failedRecord['_id'],
-            });
-          } catch (e) {
-            applicationContext.logger.info('Error', e);
-            await applicationContext.notifyHoneybadger(e);
-          }
+      const catchDeleteRecordError = failedRecord => async e => {
+        applicationContext.logger.info(
+          `processStreamRecordsInteractor job ${processJobId} deleteRecord error for record ${failedRecord['_id']}:`,
+          e,
+        );
+        await applicationContext.notifyHoneybadger(e);
```

Comment:
Can we also log this with the `logger` at the error level?

Reply:
Updated!